sdk/python/kfp/client/auth.py

```diff
@@ -179,40 +190,149 @@ def get_google_open_id_connect_token(service_account_credentials):
     return token_response['id_token']
 
 
-def get_refresh_token_from_client_id(client_id, client_secret):
+def get_refresh_token_from_client_id(client_id: str, client_secret: str):
```

Comment:
ditto: return annotation, and for functions down below

Reply:
Done.